lib/bamboozled/api/base.rb

```diff
@@ -60,7 +60,7 @@ def request(method, path, options = {})
           when 502
             raise Bamboozled::GatewayError.new(response, params, 'The load balancer or web server had trouble connecting to the Bamboo app. Please try the request again.')
           when 503
-            raise Bamboozled::ServiceUnavailable.new(response, params, 'The service is temporarily unavailable. Please try the request again.')
+            raise Bamboozled::ServiceUnavailable.new(response, params, 'The bamboo service is refusing requests. Please try again later.')
```

Comment:
Perhaps change to "The BambooHR directory is refusing this request due to rate limiting.  Please try again later."